executor/executor.go

```diff
@@ -592,7 +623,8 @@ func (e *HashJoinExec) Fields() []*ast.ResultField {
 
 var batchSize = 128
 
-// Worker to get big table rows.
+// fetchBigExec fetches rows from big table in one background goroutine
```

Comment:
big table in one background goroutine -> the big table in a background goroutine